docs/os/build-details.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Build details
+parent: Operating System
+nav_order: 210
+---
+## Armbian build details
+
+This document goes into more detail on how the Armbian build process works.
+
+The [`armbian/build.sh`](https://github.com/digitalbitbox/bitbox-base/blob/master/armbian/build.sh) script is the entrypoint to the Armbian build process.
```

Comment:
To have a link from the previous chapter (Building the Armbian base image) that only references the `make` command, we could note that `make` calls the `build.sh` script.

Reply:
Sure, added.